[lua, sql] Adjust and simplify Guivre behavior

loginosXI opened the pull request:
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Adjusts Guivre's detection so that he no longer aggros via sound and is sight only. He will also always aggro, even if 99.
Moves lua mob mods/mods to mob_family_mods.sql

Capture: [1](https://youtu.be/WjkbHITRm84?t=55)

## Steps to test these changes

!zone Kuftal Tunnel
!spawnmob 17490234

And attempt to aggro Guivre whilst invisible and without sneak.

loginosXI committed:
Adjusts Guivre's aggro behavior

Guivre does not aggro via sound, and will always aggro players regardless of level.

Simplify Guivre parameters

Moves mob mods/mods to sql from lua.

loginosXI committed:
Adjusts Guivre's aggro behavior

Guivre does not aggro via sound, and will always aggro players regardless of level.

loginosXI committed:
Simplify Guivre parameters

Moves mob mods/mods to sql from lua.

Xaver-DaRed commented:
You are moving modifiers in the exact opposite direction we want them moved.
Revert that

Xaver-DaRed reviewed (changes requested):
Revert the changes to modifiers.

Instead, move the modifiers from sql to lua

loginosXI committed:
Undo removal of lua behaviors, cleanup sql instead

Re-applys mob mods in lua as per direction, and removes Wyvern-Guivre from mob_family_mods

loginosXI committed:
Merge branch 'Guivre_behavior' of https://github.com/loginosXI/baseLSB into Guivre_behavior
